osu.Game.Rulesets.Mania/Skinning/Legacy/LegacyBodyPiece.cs

```diff
@@ -92,7 +95,8 @@ private void load(ISkinSource skin, IScrollingInfo scrollingInfo, DrawableHitObj
                 d.RelativeSizeAxes = Axes.Both;
                 d.Size = Vector2.One;
                 d.FillMode = FillMode.Stretch;
-                // Todo: Wrap
+                d.Height = minimumColumnWidth / d.DrawWidth * 1.6f;
```

Comment:
What is this `1.6f`?

Reply:
Dunno. It's what stable had, and seemed to visually match closer than not having it.

For reference:

![JetBrains Rider-EAP 2023-01-25 at 04 14 58](https://user-images.githubusercontent.com/191335/214478435-cbb572b9-82aa-46d9-a765-e72ce2feb511.png)

https://github.com/peppy/osu-stable-reference/blob/1531237b63392e82c003c712faa028406073aa8f/osu!/GameplayElements/HitObjects/Mania/HitCircleMania.cs#L68-L74

Reply:
I've added a brief inline comment mentioning that this comes from stable (https://github.com/ppy/osu/pull/22389/commits/34aa8b872eb38a460b2a726d2f689a5da96c80c2).